screens/Checkout.tsx

```diff
@@ -55,38 +52,77 @@ const Checkout: React.FC<Props<'Checkout'>> = ({navigation}) => {
   const checkoutUrl = `${baseURL}?${qs}`;
 
   return (
-    <Modal
-      animationType="slide"
-      transparent={true}
-      visible={modalVisible}
-      onRequestClose={() => {
-        Alert.alert('Modal has been closed.');
-        setModalVisible(!modalVisible);
-        navigation.navigate('Home');
-      }}>
-      <WebView
-        source={{uri: checkoutUrl}}
-        domStorageEnabled={true}
-        javaScriptEnabled={true}
-        javaScriptEnabledAndroid={true}
-        style={{flex: 1}}
-        onNavigationStateChange={event => {
-          const url = event.url;
-          const Params = url.split('?')[1];
-          const parsed = queryString.parse(Params);
-          if (parsed.action === 'cancelled') {
-            Alert.alert('Payment Cancelled!');
-            setModalVisible(!modalVisible);
-            navigation.navigate('Home');
-          }
-          if (parsed.action === 'complete') {
-            Alert.alert('Payment Successfull');
-            setModalVisible(!modalVisible);
-            navigation.navigate('Home');
-          }
-        }}
-      />
-    </Modal>
+    <>
+      <View style={styles.view}>
+        <Text style={styles.text}>Checkout page SafePay</Text>
```

Comment:
the component we will expose will not have this text  included right?

Reply:
yup!